imcsdk/apis/admin/ntp.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2016 Cisco Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+This module implements all the ntp related functionality
+"""
+from imcsdk.mometa.comm.CommNtpProvider import CommNtpProvider
+from imcsdk.imccoreutils import _is_valid_arg
+
+
+def _get_ntp_mo(handle=None, from_server=False):
```

Comment:
I wanted to put it in a single place, so that if the dn/rn changes, we need to change only at a single location. 
We can actually remove the boolean and pass only the handle to simplify it.